binder/environment.yml

```diff
@@ -7,9 +7,8 @@ dependencies:
   - dask=2.20.0
   - dask-image=0.2.0
   - dask-ml=1.6.0
-  - dask-labextension=2.0.2
-  - jupyterlab=2.1
-  - nodejs=16
+  - dask-labextension>=5.2.0
+  - jupyterlab>=3
```

Comment:
This introduces some conflicts with #218, so this should be merged first then I can fix that one.